conditions.go

```diff
@@ -79,6 +79,13 @@ func GenerateConditions(ctx Context, md []ConditionMetadata) []models.Condition
 		}
 	}
 
+	afibChance := rand.Intn(100)
+	// 3% Chance of AFib in the general population according to Chris [citation needed]
```

Comment:
It's actually 2% of the population under 65, and 9% of the population 65+.  Source: http://www.cdc.gov/dhdsp/data_statistics/fact_sheets/fs_atrial_fibrillation.htm